build/win32/code.iss

```diff
@@ -1413,7 +1432,11 @@ function PrepareToInstall(var NeedsRestart: Boolean): String;
 begin
   if IsNotBackgroundUpdate() then
     StopTunnelServiceIfNeeded();
-  Result := ''
+
+  if IsNotBackgroundUpdate() and not StopTunnelOtherProcesses() then
+     Result := '{#NameShort} is still running a tunnel process. Please stop the tunnel before installing.'
+  else
+  	Result := '';
```

Comment:
I don't have a ton of context on this code, but do we have the ability to restart the stopped tunnel processes after the update is done?

Reply:
We do, and do so automatically, if they're running `code tunnel`. But there are some other commands like `code tunnel service log` that we prefer to kill and don't have a way of restoring, which is what the original issue was about. This catches these.